parseAndPopulate/groupings.py

```diff
@@ -490,14 +490,10 @@ def parse_and_load(self):
                 self._parse_raw_capability(module.text)
             modules = self.root.iter('{}capability'.format(tag.split('hello')[0]))
 
-        try:
-            schema_parts = SchemaParts(
-                repo_owner=self.repo_owner, repo_name=self.repo_name,
-                commit_hash=self.commit_hash, submodule_name=self.submodule_name)
-        except:
-            LOGGER.exception('Missing attribute, likely caused by a broken path in {}/platform-metadata.json'
```

Comment:
Are you sure this can be safely removed? If [this](https://github.com/YangCatalog/backend/blob/80c4b2b328f47b725179545132674af40c4d2fda/parseAndPopulate/groupings.py#L389) check doesn't pass, I think the attributes will remain uninitialized.

Reply:
Yeah, I didn't see it, fixed, thanks